internal-services/catalog/create-advisory-task.yaml

```diff
@@ -58,19 +58,20 @@ spec:
               key: git_author_email
       script: |
           #!/usr/bin/env sh
-          set -ox pipefail
+          set -eox pipefail
 
-          # This should get overwritten if any command fails
-          echo -n "Success" > $(results.result.path)
-
-          failfunc() {
+          exitfunc() {
               local err=$1
               local line=$2
               local command="$3"
-              echo -n "$0: ERROR '$command' failed at line $line - exited with status $err" > $(results.result.path)
+              if [ $err -eq 0 ] ; then
+                  echo -n "Success" > $(results.result.path)
+              else
+                  echo -n "$0: ERROR '$command' failed at line $line - exited with status $err" > $(results.result.path)
+              fi
               exit 0 # exit the script cleanly as there is no point in proceeding past an error
```

Comment:
If we go with catching EXIT, this comment should be updated - it is now called on either successful finish or on a failure.

Reply:
will update

Reply:
fixed